guides/source/testing.md

```diff
@@ -546,17 +546,17 @@ class UserFlowsTest < ActionDispatch::IntegrationTest
 end
 ```
 
-Inheriting from `ActionDispatch::IntegrationTest` comes with some advantages. This makes available some additional helpers to use in your integration tests.
+Here the test is inheriting from `ActionDispatch::IntegrationTest`. This makes available some additional helpers to use in our integration tests.
```

Comment:
This makes some additional helpers available for us to use in our integration tests.